packages/turf-rectangle-grid/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@turf/rectangle-grid",
+  "version": "6.0.2",
+  "description": "turf rectangle-grid module",
+  "main": "main.js",
+  "module": "main.es.js",
+  "types": "index.d.ts",
+  "files": [
+    "index.js",
+    "index.d.ts",
+    "main.js",
+    "main.es.js"
+  ],
+  "scripts": {
+    "pretest": "rollup -c ../../rollup.config.js",
+    "test": "node -r @std/esm test.js",
+    "posttest": "node -r @std/esm ../../scripts/validate-es5-dependencies.js",
+    "bench": "node -r @std/esm bench.js",
+    "docs": "node ../../scripts/generate-readmes"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/Turfjs/turf.git"
+  },
+  "keywords": [
+    "turf",
+    "grid",
+    "regular",
+    "cartesian",
+    "grid"
+  ],
+  "author": "Turf Authors",
+  "contributors": [
+    "Steve Bennett <@stevage>",
+    "Adam Michaleski <@adam3039>"
+  ],
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Turfjs/turf/issues"
+  },
+  "homepage": "https://github.com/Turfjs/turf",
+  "devDependencies": {
+    "@turf/bbox-polygon": "*",
+    "@turf/truncate": "*",
+    "@std/esm": "*",
+    "benchmark": "*",
+    "rollup": "*",
+    "write-json-file": "*",
+    "load-json-file": "*",
+    "tape": "*"
+  },
+  "dependencies": {
+    "@turf/helpers": "*"
```

Comment:
`@turf/boolean-intersects`, `@turf/distance` and `@turf/invariant` are missing

Reply:
Done